src/js/components/DataChart/Detail.js

```diff
@@ -172,12 +183,15 @@ const Detail = ({
                 onMouseOver={(event) => {
                   activeIndex.current = event.currentTarget;
                   setDetailIndex(i);
+                  announce(getContent(i), 'assertive');
                 }}
                 onMouseLeave={onMouseLeave}
                 onFocus={() => {}}
                 onBlur={() => {}}
               >
                 <Box
+                  role="img"
+                  aria-label={getContent(i)}
```

Comment:
VoiceOver announces empty list items as 'empty group'. This works around that by adding the content for each detail as a child of each list item.